### PR TITLE
Fix #31, #23 and makes other small improvements

### LINKED ISSUE #31: Histogram does not updates when xAccessor/yAccessor props change.
I have several metrics in each `data` datapoint and I want to change the `yAccessor` so that I can change between the metrics.  
When I do so the Histogram does not update but if I brush it will reflect the new metric.

The Histogram's `componentDidUpdate(prevProps)` checks if the data has changed but **it does not takes in account that the accessors may have change** thus resulting in different data being rendered.

### PR DESCRIPTION
**Pull request by lfac-pt**
- Allow the user to define a custom `brushDomain` via `props`.
- Upgrades Storybook to 5.x
- The scales are updared if `yAccessor` or `xAccessor` are changed
- Update eslint config

### CODE REVIEW

**Comment by nancruz** [comment]
@lfac-pt in UIKit project, we are taking advantage of our CI to build the documentation and publish it. Can you have the same approach here?

**Comment by lfac-pt** [comment]
I'd say this is something to investigate in the future, for tomorrow it's too risky to add this

**Comment by nancruz** [comment]
Can we add an issue to keep track of it?

File: src/Histogram/Histogram.js [hunk 1]
```diff
@@ -422,16 +424,16 @@ export class Histogram extends PureComponent {
      */
 
     _renderBarTooltip(currentBar) {
+        if (typeof this.props.tooltipBarCustomization !== "function") {
```
[the remark this answers is not in the document]
Comment (lfac-pt) [reply src/Histogram/Histogram.js]: I want the histogram to have as few dependencies as possible

File: README.md [hunk 2]
```diff
@@ -65,6 +67,11 @@ This callback will be called when the selected intervall changes.
 
 To render a tooltip when the mouse hovers it this prop should be passed with a function that returns a React Element. This function will receive as an argument the data object relative to that column.
 
+## `brushDomain`
+**Type** `Object` **Default** `undefined`
+A custom brush domain. Should have two properties: `min` and `max` both timestamps. Please note that if the user
```
[the remark this answers is not in the document]
Comment (lfac-pt) [reply README.md]: A timestamp is usually a number in milliseconds

[the remark this answers is not in the document]
Comment (lfac-pt) [reply README.md]: But I can clarify

[the remark this answers is not in the document]
Comment (nancruz) [reply README.md]: Yes, it should be more clear

**Pull request merged**